rllib/utils/pre_checks/env.py

```diff
@@ -303,47 +325,69 @@ def check_base_env(env: "BaseEnv") -> None:
         )
         raise ValueError(error)
 
-    _check_reward(reward, base_env=True)
-    _check_done(done, base_env=True)
-    _check_info(info, base_env=True)
+    _check_reward(reward, base_env=True, agent_ids=env.get_agent_ids())
+    _check_done(done, base_env=True, agent_ids=env.get_agent_ids())
+    _check_info(info, base_env=True, agent_ids=env.get_agent_ids())
 
 
-def _check_reward(reward, base_env=False):
+def _check_reward(reward, base_env=False, agent_ids=None):
     if base_env:
         for _, multi_agent_dict in reward.items():
-            for _, rew in multi_agent_dict.items():
-                assert isinstance(rew, (float, int)), (
-                    "Your step function must return a rewards that are"
-                    f" integer or float. reward: {rew}"
+            for agent_id, rew in multi_agent_dict.items():
+                assert (
```

Comment:
Can we make these non-assertions as well, e.g. ValueError?

Reply:
done